monitoring-support/src/main/java/org/terracotta/monitoring/PlatformClientFetchedEntity.java

```diff
@@ -36,13 +35,13 @@
 
   public String clientIdentifier;
   public String entityIdentifier;
-  public transient ClientDescriptor clientDescriptor;
+  public transient Object clientDescriptor;
```

Comment:
Yes we do rely on this field by direct access, and, moreover, we rely on its type also. Why this change ?